tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -742,7 +742,7 @@
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
-        padding-top: 16px;
```

Comment:
Oh, I think the more robust solution is to use `align-items: center` on flex container and remove all these padding/margins.